Intersect.Server/Migrations/Logging/20210127004452_ChatLogging.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
```

Comment:
After updating the model as requested, delete this migration and regenerate it to fix any issues here.